Tests/ConsoleKitPerformanceTests/ConsoleLoggerPerformanceTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ConsoleLoggerPerformanceTests.swift
+//  
+//
+//  Created by Cole Kurkowski on 8/19/23.
+//
+
+import ConsoleKit
+import Logging
+import XCTest
+
+final class TestConsole: Console {
+	var lastOutput: String? = nil
+	var userInfo = [AnyHashable: Any]()
+	
+	func input(isSecure: Bool) -> String {
+		""
+	}
+
+	func output(_ text: ConsoleText, newLine: Bool) {
+		self.lastOutput = text.description + (newLine ? "\n" : "")
```

Comment:
Try the test without this line - without the additional allocations it implies the numbers might be even more interesting.

Reply:
main: 0.356 seconds
logger-fragment: 0.483 seconds